content/en/Methodologies/_index.md

```diff
@@ -30,6 +30,7 @@ asset.
 - [External Network](/methodologies/external-network/)
 - [Cloud](/methodologies/cloud/)
 - [Azure AD](/methodologies/azure-ad/)
+- [Desktop] (/methodologies/desktop/)
```

Comment:
I think the space btwn ] and ( is causing this not to be rendered as a link, but rather plain text